chips/nrf52/src/ficr.rs

```diff
@@ -144,6 +146,7 @@ register_bitfields! [u32,
         /// Part Variant, Hardware version and Production configuration, encoded as ASCII
         // Note, some of these are not present in datasheets
         // but are in nrf52.svd or are observed in the wild
+        #[allow(clippy::enum_clike_unportable_variant)]
```

Comment:
I think we should simply remove all of these from this PR and resolve this in #1823. I know that means this PR doesn't _quite_ get clippy passing, but it's also been through a lot of churn at this point, so it'd be nice to land all of the uncontroversial stuff. I think without these, it's ready to go.

Reply:
Yeah of course, sorry I meant to include that change originally